apps/graphql/src/definitions/Team.ts

```diff
@@ -30,7 +30,7 @@ export const typeDefs = gql`
     periodEndDate: DateTime
     plan: Plan
     purchase: Purchase
-    purchaseStatus: PurchaseStatus!
+    purchaseStatus: PurchaseStatus
```

Comment:
Il will break the app because we assume that the PurchaseStatus is not null when the account is a Team: https://github.com/argos-ci/argos/blob/f97fe2c0ce5e652495c5b7c3bc7b25084c5f0176/apps/app/src/containers/PaymentBanner.tsx#L199

Reply:
I fixed it.